README.rst

```diff
@@ -33,7 +33,31 @@ You can also set the `TESTING` flag to `True` so no mails are sent
 Deploy it
 =========
 
-To deploy it, I'm using gunicorn and supervisord.
```

Comment:
The Maybe add some introduction so that  the reader understands she is facing an alternative. Something like 

> You have numerous options to deploy ihatemoney. Two of them are documented at the moment: